### ISSUE TITLE
Extension 'ms-toolsai.jupyter' CANNOT use API proposal: `notebookCellExecutionState` 

### ISSUE THREAD

**Issue by kolserdav** [microsoft/vscode-jupyter#13237]
# Behaviour
## Expected vs. Actual

XXX

## Steps to reproduce:

1. XXX

# Diagnostic data

-   Python version (& distribution if applicable, e.g. Anaconda): 3.10.10
-   Type of virtual environment used (e.g. conda, venv, virtualenv, etc.): Global
-   Value of the `python.languageServer` setting: Default
- VSCode Version: 1.76.2
Commit: ee2b180d582a7f601fa6ecfdad8d9fd269ab1884
Date: 2023-03-16T10:00:00.439Z
Electron: 19.1.9
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Linux x64 6.2.8
Sandboxed: No

<details>

<summary>Output for <code>Python</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Python</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

<details>

<summary>User Settings</summary>

<p>

```

languageServer: "Jedi"

```

</p>
</details>

<details>

<summary>Full log by start</summary>

<p>

```sh
workbench.desktop.main.js:747 Extension activation failed, run the 'Developer: Toggle Developer Tools' command for more information.
workbench.desktop.main.js:615   ERR [Extension Host] extension activation failed Error: Extension 'ms-toolsai.jupyter' CANNOT use API proposal: notebookCellExecutionState.
Its package.json#enabledApiProposals-property declares:  but NOT notebookCellExecutionState.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-toolsai.jupyter
	at b (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:92:44827)
	at Mc.onDidChangeNotebookCellExecutionState (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:117:51922)
	at new PC (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:433687)
	at f (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:544291)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546181
	at t.resolveInstance (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546208)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549240
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549577
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548900
	at y (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548932)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548794
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548728
	at Array.map (<anonymous>)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548700
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545539
	at Array.reduce (<anonymous>)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545511
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545838
	at t.resolveInstance (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546208)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549240
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549577
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548900
	at y (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548932)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548794
	at t.resolve (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:550764)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519737
	at e._get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519106)
	at e._getButThrowIfAsync (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519166)
	at e.get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:514861)
	at Ly.get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:69994)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:552800
	at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
workbench.desktop.main.js:90 [Extension Host] extension activation failed Error: Extension 'ms-toolsai.jupyter' CANNOT use API proposal: notebookCellExecutionState.
Its package.json#enabledApiProposals-property declares:  but NOT notebookCellExecutionState.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-toolsai.jupyter
	at b (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:92:44827)
	at Mc.onDidChangeNotebookCellExecutionState (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:117:51922)
	at new PC (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:433687)
	at f (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:544291)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546181
	at t.resolveInstance (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546208)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549240
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549577
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548900
	at y (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548932)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548794
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548728
	at Array.map (<anonymous>)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548700
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545539
	at Array.reduce (<anonymous>)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545511
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545838
	at t.resolveInstance (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546208)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549240
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549577
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548900
	at y (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548932)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548794
	at t.resolve (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:550764)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519737
	at e._get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519106)
	at e._getButThrowIfAsync (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519166)
	at e.get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:514861)
	at Ly.get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:69994)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:552800
	at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
T	@	workbench.desktop.main.js:90
workbench.desktop.main.js:90 [Extension Host] rejected promise not handled within 1 second: Error: Extension 'ms-toolsai.jupyter' CANNOT use API proposal: notebookCellExecutionState.
Its package.json#enabledApiProposals-property declares:  but NOT notebookCellExecutionState.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-toolsai.jupyter
workbench.desktop.main.js:90 [Extension Host] stack trace: Error: Extension 'ms-toolsai.jupyter' CANNOT use API proposal: notebookCellExecutionState.
Its package.json#enabledApiProposals-property declares:  but NOT notebookCellExecutionState.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-toolsai.jupyter
	at b (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:92:44827)
	at Mc.onDidChangeNotebookCellExecutionState (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:117:51922)
	at new PC (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:433687)
	at f (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:544291)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546181
	at t.resolveInstance (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546208)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549240
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549577
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548900
	at y (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548932)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548794
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548728
	at Array.map (<anonymous>)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548700
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545539
	at Array.reduce (<anonymous>)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545511
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545838
	at t.resolveInstance (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546208)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549240
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549577
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548900
	at y (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548932)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548794
	at t.resolve (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:550764)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519737
	at e._get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519106)
	at e._getButThrowIfAsync (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519166)
	at e.get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:514861)
	at Ly.get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:69994)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:552800
	at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
workbench.desktop.main.js:1979 [[object Object]]Extension 'ms-toolsai.jupyter' CANNOT use API proposal: notebookCellExecutionState.
Its package.json#enabledApiProposals-property declares:  but NOT notebookCellExecutionState.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-toolsai.jupyter
workbench.desktop.main.js:1979 Error: Extension 'ms-toolsai.jupyter' CANNOT use API proposal: notebookCellExecutionState.
Its package.json#enabledApiProposals-property declares:  but NOT notebookCellExecutionState.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-toolsai.jupyter
	at b (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:92:44827)
	at Mc.onDidChangeNotebookCellExecutionState (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:117:51922)
	at new PC (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:433687)
	at f (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:544291)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546181
	at t.resolveInstance (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546208)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549240
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549577
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548900
	at y (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548932)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548794
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548728
	at Array.map (<anonymous>)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548700
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545539
	at Array.reduce (<anonymous>)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545511
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:545838
	at t.resolveInstance (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:546208)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549240
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:549577
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548900
	at y (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548932)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:548794
	at t.resolve (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:550764)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519737
	at e._get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519106)
	at e._getButThrowIfAsync (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:519166)
	at e.get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:2:514861)
	at Ly.get (/home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:69994)
	at /home/kol/.vscode-oss/extensions/ms-toolsai.jupyter-2023.3.100-universal/out/extension.node.js:24:552800
	at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
workbench.desktop.main.js:615  INFO [perf] Render performance baseline is 40ms
workbench.desktop.main.js:90 [Extension Host] rejected promise not handled within 1 second: Error: Extension 'ms-python.python' CANNOT use API proposal: testObserver.
Its package.json#enabledApiProposals-property declares:  but NOT testObserver.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-python.python
workbench.desktop.main.js:90 [Extension Host] stack trace: Error: Extension 'ms-python.python' CANNOT use API proposal: testObserver.
Its package.json#enabledApiProposals-property declares:  but NOT testObserver.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-python.python
	at b (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:92:44827)
	at get testResults [as testResults] (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:117:33156)
	at E.activate (/home/kol/.vscode-oss/extensions/ms-python.python-2023.4.1-universal/out/client/extension.js:2:783139)
	at /home/kol/.vscode-oss/extensions/ms-python.python-2023.4.1-universal/out/client/extension.js:2:2386
	at Array.map (<anonymous>)
	at f.activateWorkspace (/home/kol/.vscode-oss/extensions/ms-python.python-2023.4.1-universal/out/client/extension.js:2:2376)
	at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
workbench.desktop.main.js:1979 [[object Object]]Extension 'ms-python.python' CANNOT use API proposal: testObserver.
Its package.json#enabledApiProposals-property declares:  but NOT testObserver.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-python.python
workbench.desktop.main.js:1979 Error: Extension 'ms-python.python' CANNOT use API proposal: testObserver.
Its package.json#enabledApiProposals-property declares:  but NOT testObserver.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-python.python
	at b (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:92:44827)
	at get testResults [as testResults] (/usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:117:33156)
	at E.activate (/home/kol/.vscode-oss/extensions/ms-python.python-2023.4.1-universal/out/client/extension.js:2:783139)
	at /home/kol/.vscode-oss/extensions/ms-python.python-2023.4.1-universal/out/client/extension.js:2:2386
	at Array.map (<anonymous>)
	at f.activateWorkspace (/home/kol/.vscode-oss/extensions/ms-python.python-2023.4.1-universal/out/client/extension.js:2:2376)
	at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
```

</p>

</details>

**Comment by amunger** [comment]
which version of the jupyter extension are you using?

**Comment by kolserdav** [comment]
> which version of the jupyter extension are you using?

v2023.3.100

**Comment by amunger** [comment]
Can you try to update vscode to the latest release (1.77)?
The engine version requirement should have been updated, but it seems like you may have a version mismatch

**Comment by ManonMarchand** [comment]
I have the same issue with this version of code :

Version: 1.77.0
Commit: 7f329fe6c66b0f86ae1574c2911b681ad5a45d63
Date: 2023-03-30T20:39:46.329Z
Electron: 19.1.9
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Linux x64 6.1.21-1-MANJARO
Sandboxed: No

And the extension in version v2023.3.100

**Comment by ManonMarchand** [comment]
Solved in v1.77.1 from the AUR instead of community code package

**Comment by kolserdav** [comment]
> Solved in v1.77.1 from the AUR instead of community code package

But `CodeOSS` has not been updated since March 2022 https://aur.archlinux.org/packages/code-git

Maybe you installed `Proprietary` version from https://aur.archlinux.org/packages/visual-studio-code-bin ?

**Comment by ManonMarchand** [comment]
Yes, exactly, I had to switch to the proprietary version
